content/en/docs/instrumentation/cpp/getting-started-new.md

```diff
@@ -0,0 +1,388 @@
+not using Oat++, that's OK - you can use OpenTelemetry C++ with any other web frameworks as well.
+
+## Dependencies 
+
+**To begin, we will install Oatpp locally using the [source code](https://github.com/oatpp) and `make`, following these general steps:**
+
+1. Download oatpp source code:
+
+Obtain the oatpp source code. You can either download the source code from the oatpp Github repository or clone it using Git.
+
+```bash 
+git clone https://github.com/oatpp/oatpp.git
+```
+
+2. Navigate to the oatpp directory:
+
+Access the oatpp directory in your terminal.
+
+```bash 
+cd oatpp
+```
+
+3. Build oatpp:
+
+Build oatpp using the `make` command. This command will trigger the build process specified in the Makefile included in the oatpp source code.
+
+```bash
+make
+```
+4. Install oatpp:
+
+This command will install the built oatpp library and headers on your system, making it accesible for development in your project. 
+
+```bash
+make install
+```
+
+**Next, install and build [OpenTelemetry C++](https://github.com/open-telemetry/opentelemetry-cpp) locally using CMake, following these general steps:**
+
+1. Clone the OpenTelemetry C++ repository:
+
+Clone the OpenTelemetry C++ GitHub repository to your local machine.
+
+```bash
+git clone https://github.com/open-telemetry/opentelemetry-cpp.git
+```
+
+2. Navigate to the OpenTelemetry C++ SDK directory:
+
+Change your working directory to the OpenTelemetry C++ SDK directory.
+
+```bash
+cd openTelemetry-cpp
+```
+
+3. Create a build directory:
+
+It's a good practice to create a seperate directory for the build files to keep your source directory clean.
+
+```bash
+mkdir build
+cd build 
+```
+
+4. Configure and generate the build system:
+
+Run the CMake from the build directory to configure the build.
+
+```bash
+cmake ..
+```
+
+5. Build the project:
+
+After configuring the build system, execute the build process.
+
+```bash
+cmake --build .
+```
+
+**Finally, create a new project directory called `roll-dice`.**
+
+## Create and launch an HTTP Server
+
+**Integrate oatpp in your project:**
+
+In your `roll-dice` project, utilize the oatpp library by referencing the oatpp headers and linking them when compiling your project.. 
+
+Create a file called `CMakeLists.txt` to define the oatpp library directories, include paths, and link against oatpp during the compilation process.
+
+Here is what `CMakeLists.txt` might look like:
+
+```cmake
+cmake_minimum_required(VERSION 3.1)
+
+# Set C++ standard (e.g., C++17)
+set(CMAKE_CXX_STANDARD 17)
+
+set(project_name my-oatpp-project)
+
+# Define your project's source files
+set(SOURCES
+    main.cpp  # Add your source files here
+)
+
+# Create an executable target
+add_executable(myapp ${SOURCES})
+
+set(OATPP_ROOT /path/to/oatpp)
+set(OATPP_LIB /path/to/oatpp/lib)
+
+#set the path to the directory containing "oatpp" package configuration files 
+include_directories(${OATPP_ROOT}/src)
```

Comment:
are you sure about the /src? in my case this is not required

Reply:
You're correct; technically, it would work without /src. However, since the source files we're working with are inside /src, it's best to navigate to that directory.